resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -748,6 +756,18 @@ export default {
             }
         }
 
+        Events.$on('focusTypeaheadEvent', () => { 
+            this.focusInputFindlabel = false;
+            this.$nextTick(() => {
+                this.focusInputFindlabel = true;
+            });
+           this.focusInputFindlabel = false;
+        });
+        
+        Keyboard.on('control+k', () => {
+            this.openSidebarLabels()
+        });
+
```

Comment:
```suggestion
        Keyboard.on('control+k', this.openSidebarLabels, 0, this.listenerSet);
```
This does the same as your code but is more consistent with the other keyboard listeners.